build.sbt

```diff
@@ -56,10 +56,11 @@ releaseProcess := Seq(
 )
 
 resolvers += "Guardian GitHub Repository" at "https://guardian.github.io/maven/repo-releases"
+resolvers ++= Resolver.sonatypeOssRepos("snapshots")
 
 libraryDependencies ++= Seq(
   "com.gu" %% "content-api-models-scala" % "17.3.0",
-  "com.gu" %% "thrift-serializer" % "5.0.0",
+  "com.gu" %% "thrift-serializer" % "5.0.2-SNAPSHOT",
```

Comment:
Presumably this will be updated when the full version is published?

Reply:
:+1 yep